<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWizardPage</class>
 <widget class="QWizardPage" name="ServerWizardPage">
  <property name="title">
   <string>Server details</string>
  </property>
  <property name="subTitle">
   <string>Details used to connect to IRC</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&amp;Host:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditHost</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditHost">
     <property name="whatsThis">
      <string>Entering a server host is mandatory. The connection is established to the given server host and port.</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNoAutoUppercase|Qt::ImhUrlCharactersOnly</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>&amp;Port:</string>
     </property>
     <property name="buddy">
      <cstring>spinBoxPort</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSpinBox" name="spinBoxPort">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="whatsThis">
        <string>Entering a server port is mandatory. The connection is established to the given server host and port.</string>
       </property>
       <property name="minimum">
        <number>1024</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>6667</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxSecure">
       <property name="whatsThis">
        <string>If this check box is checked, a secure connection is established to the given server.</string>
       </property>
       <property name="text">
        <string>&amp;Secure</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Pass&amp;word:</string>
     </property>
     <property name="buddy">
      <cstring>lineEditPass</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditPass">
     <property name="whatsThis">
      <string>Entering a server password is optional. Some servers are secured with a password.</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText</set>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>lineEditPass</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>lineEditHost</zorder>
  <zorder>label_4</zorder>
 </widget>
 <tabstops>
  <tabstop>lineEditHost</tabstop>
  <tabstop>spinBoxPort</tabstop>
  <tabstop>lineEditPass</tabstop>
  <tabstop>checkBoxSecure</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
